src/shared/components/layouts/styles/sidebar-layout.module.scss

```diff
@@ -0,0 +1,63 @@
+@import 'franklin-sites/src/styles/common/z-index';
+@import 'franklin-sites/src/styles/colours';
+@import 'franklin-sites/src/styles/mixins';
+@import '../../../styles/settings';
+@import '../../../styles/mixins';
+
+.sidebar-layout {
+  --sidebar-size: 18ch;
+  @include fs-breakpoints('medium') {
+    --sidebar-size: 25ch;
+  }
+  @include fs-breakpoints('large') {
+    --sidebar-size: 30ch;
+  }
+
+  max-width: 100%;
+  min-height: calc(100vh - var(--top-header-margin));
+  display: grid;
+  align-items: start;
+  grid-template-columns: 3rem calc(var(--sidebar-size) - 3rem) minmax(0, 1fr);
```

Comment:
Uses 3 columns (2 for sidebar, 1 for main content), even though visually there are 2, in order to possibly have in the future the main content cover the sidebar partially if we want to have a triggered "expanded" view. Not so much of a problem for results pages because we have the scroll, but for entries where we force no scroll then it might get useful